components/RouteForm/StyledRouteForm.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+const Form = styled.form`
+  position: absolute;
+  top: 2rem;
+  left: 50%;
+  transform: translate(-50%);
+  z-index: 1;
+  background-color: white;
+  display: ${(props) => (props.savedRoute ? "flex" : "none")};
+  flex-direction: column;
```

Comment:
whenever display:none; is applied, also having a flex-direction does not make a lot of sense. You could consider making both lines conditional:

```suggestion
display: none;
${(props) =>
    props.savedRoute &&
    css`
      display: flex;
      flex-direction: column;
    `}
```

don't forget to import { css } for this to work properly